packages/components/avatar/src/avatar.ts

```diff
@@ -194,12 +179,23 @@ export class Avatar extends LitElement {
   }
 
   #renderPicture(): TemplateResult {
-    const { x, y, containerSize, radius, size } = this.image!;
+    const { x, y, containerSize, radius, size } = this.image!,
+      parts: string[] = [];
```

Comment:
`parts` isn't a very clear name (i assume it was called this in a function/context that made the label, so it could be inferred from the context that it were label-parts, but that context is gone here.)